src/tests/test_jsproxy.py

```diff
@@ -2249,8 +2250,8 @@ def test_python_reserved_keywords(selenium):
     assert o.async_ == 1
     assert o.async__ == 2
     assert o.async___ == 3
-    assert getattr(o, "async_") == 1
-    assert getattr(o, "async__") == 2
+    assert o.async_ == 1
+    assert o.async__ == 2
```

Comment:
```suggestion
    assert getattr(o, "async_") == 1
    assert getattr(o, "async__") == 2
```
% `ruff rule B009`
# get-attr-with-constant (B009)

Derived from the **flake8-bugbear** linter.

Autofix is always available.

Message formats:
* Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.